### PR TITLE
corrections to simplify codes

### PR DESCRIPTION
**Pull request by Arcziwald**
I looked through the entire project and paid attention to formatting and possible code simplifications in case of repetitiveness, etc.

### CODE REVIEW

File: src/common/PeopleList/index.js [hunk 1]
```diff
@@ -2,45 +2,25 @@ import { Main } from "../Main";
...
+              picture={person.profile_path}
+              name={person.name}
+              role={
+                moviePage ? person.character ?? person.job : person.character
+              }
+            />
+          </StyledLink>
+        </ListItem>
+      ))}
+    </List>
```
Comment (KrystianZybura) [inline src/common/PeopleList/index.js:23]: Nice, that you simplified this! 😎

File: src/common/Navigation/index.js [hunk 2]
```diff
@@ -12,27 +11,23 @@ import {
...
           <StyledNavLink
             to="/people"
-            className={
-              location.pathname === "/search" ||
-              location.pathname.includes("/person/")
-                ? "active"
-                : ""
-            }
+            className={isPeopleActive ? "active" : ""}
           >
```
Comment (KrystianZybura) [inline src/common/Navigation/index.js:31]: Great idea, looks much better now 👌

**Review by KrystianZybura — APPROVED** [review]
Great job! Code looks much better now ✌

File: src/common/Navigation/Search/index.js [hunk 3]
```diff
@@ -7,12 +7,11 @@ export const Search = () => {
   const history = useHistory();
   const [searchValue, setSearchValue] = useState("");
 
-  const placeholderText = location.pathname.includes("movie")
+  const moviePath = location.pathname.includes("movie");
+  const placeholderText = moviePath
     ? "Search for movies..."
```
Comment (WoytekMig) [inline src/common/Navigation/Search/index.js:12]: Nice ! That's smart ! :)

**Review by WoytekMig — APPROVED** [review]
mission accomplished ! :D good job ;)

**Pull request merged**